docs/configure_harmony.md

```diff
@@ -0,0 +1,65 @@
+# Configure Harmony service for availability management
+
+This guide instructs you to configure your marketplace to use [Harmony](https://www.github.com/sharetribe/harmony) service for availability management.
+
+This guide does not instruct how to install and run Harmony. See [README.md](https://github.com/sharetribe/harmony/blob/master/README.md) in Harmony repository for installation instructions.
+
+## Prerequisites
+
+* Harmony version 0.1.2 or newer
+* Sharetribe version 6.2.0 or newer
+* Harmony service up and running
+* Marketplace configured to use transaction process with PayPal payments.
+
+The availability management is tightly coupled to the transaction process, so PayPal payments need to be enabled to use availability management.
+
+## Getting started
+
+1. Make sure that Harmony is up and running
+
+  Try to open: [http://localhost:8085/apidoc/index.html](http://localhost:8085/apidoc/index.html)
+
+  If everything goes well, you should see the Swagger UI.
+
+  If you are running Harmony service in other location than `localhost:8085` (e.g. different port) you need to change the URL. `localhost:8085` is the default location.
```

Comment:
I suppose the full URL needs to be specified, together with protocol? I.e. `http://localhost:8085`.